CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 Changelog
 =========
 
+* **2017-02-03**: [BC break] Removed own SimplePager in favour of `Sonata\AdminBundle\Datagrid\SimplePager` to avoid duplications.
```

Comment:
It's not automatic, it's a human ( @Soullivaneuh ) that does this. The goal is to avoid n PRs conflicting with each other.